ARM_DisassemblerExplorer/src/utils/StringUtils.h

```diff
@@ -91,6 +93,26 @@ class StringUtils
     {
         return label + "##" + ToString(uniqueId);
     }
+
+    static std::string RemoveExtraWhitespaces(const std::string_view inputString)
+    {
+        /* Copy each character from the input string into the output string, unless it's
+         * repeating space */
+        /* From : https://stackoverflow.com/a/35302029 */
+        std::string outputString;
+
+        /* clang-format off */
+        std::unique_copy(inputString.begin(),
+                         inputString.end(),
+                         std::back_insert_iterator<std::string>(outputString),
+                         [](char a, char b)
+                         {
+                            return std::isspace(a) && std::isspace(b);
+                         });
+        /* clang-format on */
+
+        return outputString;
+    }
```

Comment:
could there be an overload of this that also takes a `const std::string&` as parameter please?

Also, I think you forgot the `&` for the `const std::string_view` ;3